docs/excel/excel-add-ins-worksheet-display.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Adjust worksheet display settings
+description: How to adjust some worksheet display settings to make reports easier to read.
+ms.date: 04/03/2025
+ms.localizationpriority: medium
+---
+
+# Adjust worksheet display settings
+
+Excel is often used for reporting scenarios where you want to share worksheet data with others. Your Office Add-in can reduce visual clutter and help focus attention by controlling the appearance of the worksheet. The Office JavaScript API supports changing several visual aspects of the worksheet.
+
+## Page layout and print settings
+
+Add-ins have access to page layout settings at a worksheet level. These control how the sheet is printed. A `Worksheet` object has three layout-related properties: `horizontalPageBreaks`, `verticalPageBreaks`, `pageLayout`.
+
+`Worksheet.horizontalPageBreaks` and `Worksheet.verticalPageBreaks` are [PageBreakCollections](/javascript/api/excel/excel.pagebreakcollection). These are collections of [PageBreaks](/javascript/api/excel/excel.pagebreak), which specify ranges where manual page breaks are inserted. The following code sample adds a horizontal page break above row **21**.
```

Comment:
I think we should avoid pluralizing code objects that aren't actually plural (PageBreakCollections, PageBreaks). We should avoid "above" per style guide. 
```suggestion
`Worksheet.horizontalPageBreaks` and `Worksheet.verticalPageBreaks` are [PageBreakCollection](/javascript/api/excel/excel.pagebreakcollection) objects. These are collections of [PageBreak](/javascript/api/excel/excel.pagebreak) objects, which specify ranges where manual page breaks are inserted. The following code sample adds a horizontal page break before row **21**.
```

Reply:
PS: Sorry you just copied this from an existing article and now I'm asking you to change it.